src/ucp/api/ucp.h

```diff
@@ -325,6 +325,8 @@ typedef struct ucp_config {
     size_t                                 bcopy_thresh;
     /** Threshold for switching UCP to rendezvous protocol */
     size_t                                 rndv_thresh;
+    /** How much packet data to dump in debug mode */
```

Comment:
this is not correct documentation. It's how much packed data to dump,not the log buffer size.